protobuf.bzl

```diff
@@ -506,3 +514,32 @@ def check_protobuf_required_bazel_version():
     copied filegroup. (Fixed in bazel 0.5.4)
     """
     versions.check(minimum_bazel_version = "0.5.4")
+
+
+def conformance_test(name, testee, failure_list = None, text_format_failure_list = None):
```

Comment:
I think we unfortunately have users importing `protobuf.bzl` right now, so it's in a pseudo-public state. Are we intending to make this public for users? If not can we move it to a separate file that is more clearly internal? (Ideally with an actual visibility list, but I'm not sure if Bazel supports this right now).

Reply:
honestly, I think we should make this public eventually.

Reply:
If we mean to make this function public, I think it makes more sense to put in `conformance/`. This file is full of cruft that we want to burn down, I don't think we should add things to it.

If we're trying to keep this function private, I think we should move it to a filename that makes that clear, and not `protobuf.bzl` which we know a lot of users already include directly.

Reply:
Talked offline, moved to internal.bzl